web/client/components/import/dragZone/enhancers/dropZoneHandlers.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {compose, withHandlers} = require('recompose');
+/**
+ * Enhancer that provides a method to open file system browser of the dropzone.
+ * @memberof components.import.dragZone.enhancers
+ * @function
+ *
+ */
+module.exports = compose(
```

Comment:
An example of splitting too much: is this enhancer useful to anything else than the dragZone? (can it be used on more than one type of component, and how?)